primitives/core/src/sr25519.rs

```diff
@@ -566,6 +579,28 @@ impl Pair {
 		let kp = mini_key.expand_to_keypair(ExpansionMode::Ed25519);
 		(Pair(kp), mini_key.to_bytes())
 	}
+
+	/// Verify a signature on a message. Returns true if the signature is good.
+	pub fn verify_format<M: AsRef<[u8]>>(format: Format, sig: &Signature, message: M, pubkey: &Public) -> bool {
```

Comment:
Why isn't that just called verify deprecated?

There will be no more "formats" and formats is also not the correct term. It was broken before and got fixed.

I would drop all this format stuff, that really isn't needed and don't brings any advantages.